gem_city.rb

```diff
@@ -0,0 +1,50 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+
+  def initialize
+    @people = {
+      thieves: 5,
+      officers: 1
+    }
+    @population = 50
+  end
+
+  def thieves(thieves_number = @people[:thieves])
```

Comment:
Officers have both a getter and setter method, but thieves just have this hybrid method. I think two methods is better.